doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,432 @@
+    * PWM120
+    * SPIM120
+    * EXMIF
+
+  * UICR validation support.
+  * MSPI EXMIF support.
+
+* Power Management:
+
+  * System-On-All-Idle power improvements.
+  * Thread/Matter power efficiency improvements.
+
+Added the following features as experimental:
+
+* Bluetooth:
+
+  * :ref:`Parallel scanning and initiating connection <bt_scanning_while_connecting>`.
+    It reduces the overall waiting time when there are several Bluetooth devices to be discovered and connected at the same time.
+
+Improved the following features:
+
+* Better BICR handling and generation.
+* Enhanced power management features across various subsystems.
+
+Implemented the following bug fixes:
+
+* NCSDK-30802: Resolved an issue where the nRF54H20 device suddenly stopped transmitting ESB packets after nrfxlib 3.9.0.
+* NCSDK-30161: Fixed an assertion during boot time caused by a combination of :kconfig:option:`CONFIG_ASSERT`, :kconfig:option:`CONFIG_SOC_NRF54H20_GPD`, and external flash.
+* NCSDK-30117: Ensured that declaring a MEM component pointing to a memory region not assigned to a particular core is no longer possible.
+* NCSDK-29682: Added support for the `cose-alg-sha-512` algorithm in the SUIT module.
+
+Limitations
+***********
+
+On the nRF54H20 SoC, The device firmware update (DFU) procedure from external flash memory does not work with the new flash memory driver for XIP.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.9.0-nRF54H20**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.9.0-nRF54H20`_.
+
+Integration test results
+************************
+
+The integration test results for this tag can be found in the following external artifactory:
+
+* `Twister test report for nRF Connect SDK v2.9.0-nRF54H20`_
+* `Hardware test report for nRF Connect SDK v2.9.0-nRF54H20`_
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.9.0-nrf54h20-rc1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.0-nRF54H20`_ for the list of issues valid for the latest release.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.9.0-nRF54H20`_ for the changes required or recommended when migrating your nRF54H Series application from |NCS| v2.8.0 to |NCS| v2.9.0-nrf54h20-rc1.
+
+.. _ncs_release_notes_290-nRF54H20-rc1_changelog:
+
+Changelog
+*********
```

Comment:
Shouldn't we add these entries of nRF54H20? - https://github.com/nrfconnect/sdk-nrf/pull/19596/files#diff-67f56811be17161268a1ce745ea8aab07d9248a3ecc14987c87685b52c1e4cde, which are there in the 2.9 release notes

Reply:
Certain entries in common with 54H and other SoC should have been duplicated in the 2.9.0 release notes, without removing them from the changelog . If they got removed, it is easy to lose track of these items. I'll doublecheck and add them if missing